### PR TITLE
Render edits

### PR DESCRIPTION
**Pull request by Koniiro**

### CODE REVIEW

File: backend/CourseParser.py
Comment (eyronjuude) [inline backend/CourseParser.py:1]: `match` did not work in render?

Comment (eyronjuude) [inline backend/CourseParser.py:1]: `match` did not work in render?

File: backend/app.py
Comment (eyronjuude) [inline backend/app.py:1]: Use the updated `server.py` in `main` and try to integrate the said file to render.

Or you could just rename the `server.py` in `main` as `app.py` — your choice.

**Review by eyronjuude — CHANGES_REQUESTED** [review]
Code is ok, but I noticed some changes in the backend that caught up my attention.

Kindly check the posted review comments in the affected files.

File: backend/CourseParser.py
Comment (Koniiro) [reply backend/CourseParser.py:1]: yes di siya nagwowork sa render apparently. like when i checked the logs. yung match yung specifically na mention niya

Comment (eyronjuude) [reply backend/CourseParser.py:1]: Okay noted.

File: backend/app.py
Comment (Koniiro) [reply backend/app.py:1]: i dont see any differences with server and app pero yung need ni render is yung app.py so ill keep it as is

Comment (eyronjuude) [reply backend/app.py:1]: Resolved from 911f21a.

**Review by eyronjuude — APPROVED** [review]

**Comment by eyronjuude** [comment]
@Koniiro can you pull this branch forcefully then check the deployment status?

Apparently it failed the necessary checks after I resolved the merge conflicts. Please acknowledge.

Thanks.

**Comment by eyronjuude** [comment]
Nevermind. Sorry for the ping. I think I made a mistake in solving merge conflicts.

Gotta try fixing the commit history once more.

**Pull request merged**